pkg/ui/react-app/src/pages/graph/PanelList.tsx

```diff
@@ -144,6 +148,11 @@ const PanelList: FC<RouteComponentProps & PathPrefixProps> = ({ pathPrefix = ''
   const [enableAutocomplete, setEnableAutocomplete] = useLocalStorage('enable-autocomplete', true);
   const [enableHighlighting, setEnableHighlighting] = useLocalStorage('enable-syntax-highlighting', true);
   const [enableLinter, setEnableLinter] = useLocalStorage('enable-linter', true);
+  const [engine, setEngine] = useLocalStorage<string>('engine', 'prometheus');
```

Comment:
Right, I think we can get the default engine from the flag API call i.e `flagRes` here, and pass it to Panel, where we render the dropdown accordingly, instead of hardcoding to prometheus.